packages/ibm-products/src/components/UserAvatar/UserAvatar.js

```diff
@@ -58,8 +58,10 @@ export let UserAvatar = React.forwardRef(
       // The component props, in alphabetical order (for consistency).
       backgroundColor,
       className,
+      name,
       /* TODO: add other props for UserAvatar, with default values if needed */
       renderIcon = defaults.renderIcon,
```

Comment:
i think this needs to be changed. currently there's no way to access the group variant without already passing the group icon, which we then import in this component. given that i think the component needs to be refactored slightly.

the user can just pass in an icon directly. `<UserAvatar renderIcon={Information />`

in the render we can check for a custom icon or provide a default one. and then we can use a size dictionary to access the sizes.

```
size,
renderIcon: Icon,
...
const sizes = {
  sm: 16,
  md: 20
}
...
{Icon ? <Icon size={sizes[size]} /> : <User />
```

Reply:
Done